Cargo.toml

```diff
@@ -7,17 +7,17 @@ authors = ["Jonathan Turner <jturner@mozilla.com>"]
 cargo = { git = "https://github.com/rust-lang/cargo" }
 derive-new = "0.3"
 env_logger = "0.4"
-languageserver-types = "0.7.0"
+languageserver-types = "0.8.0"
 log = "0.3"
 racer = "2.0.6"
-rls-analysis = "0.1"
-rls-data = "0.1"
-rls-span = { version = "0.1", features = ["serialize-serde"] }
-rls-vfs = { version = "0.2", features = ["racer-impls"] }
+rls-analysis = "0.2.1"
+rls-data = "0.3.1"
+rls-span = { version = "0.4", features = ["serialize-serde"] }
+rls-vfs = { git = "https://github.com/nrc/rls-vfs", rev = "ab5eaf3ed62fe6c5e8d2eab5afdf0a511dd51cd9", features = ["racer-impls"] }
```

Comment:
[`rls-vfs`](https://crates.io/crates/rls-vfs) containing https://github.com/nrc/rls-vfs/pull/17 has not been published yet. So I specified git rev. directly.

@nrc could you publish the new version fo rls-vfs?